Assignment5/question1.js

```diff
@@ -0,0 +1,20 @@
+/*
+    1. What is the difference between ++i and i++?
+        - ++i will increment value of i first and will then evaluate expression
+        - ++i will evaluate expression and will then increment the value of i
+    
+    
+    2. What do you think would happen if you pass an index beyond the range of the string? Or if you pass a negative index? Try it out.
+        - in both cases it will give undefined 
+
+    
+    3. Do you think JSON.stringify would work for arrays as well? What about nested objects? What happens if we pass numbers, strings, undefined, null to JSON.stringify?
+        - Yes, JSON.stringify will work in all the cases
```

Comment:
take all these things as input to JSON.stringify and console log the output. Also write o/p in comments

Reply:
done